docs/debugger/entry.rst

```diff
@@ -8,8 +8,8 @@ Entering the Debugger
 Invoking the Debugger Initially
 ===============================
 
-The simplest way to debug your program is to call run `remake -X`. Give
-the name of your Makefile:
+The simplest way to debug your program is to call run `remake -X` or
```

Comment:
Should it read "is to call run ..." or just "is to call..." or "is to run"?

Reply:
Thanks - fixed.